src/Graphics/OpenGLContext/GLSL/glsl_CombinerProgramBuilder.cpp

```diff
@@ -2731,6 +2731,12 @@ CombinerInputs CombinerProgramBuilder::compileCombiner(const CombinerKey & _key,
 	}
 
 
+	if (g_cycleType <= G_CYC_2CYCLE) {
+		ssShader << "if (uCvgXAlpha != 0) cvg *= clampedColor.a;" << std::endl;
+		ssShader << "if (uAlphaCvgSel != 0) clampedColor.a = cvg; " << std::endl;
+	}
```

Comment:
duplication. I already moved these lines to the if() clause above.

Reply:
Ah, sorry, I missed it. I guess I duplicated it when rebasing.